src/main/resources/META-INF/plugin.xml

```diff
@@ -76,19 +76,19 @@
                     icon="PluginIcons.A_PLUS_REFRESH" />
         </group>
         <group id="fi.aalto.cs.apluscourses.intellij.actions.ActionGroups.EXERCISE_ACTIONS"
-               text="A+ Exercise"
+               text="A+ Assignment"
                description="Actions for A+ modules">
             <action id="fi.aalto.cs.apluscourses.intellij.actions.SubmitExerciseAction"
                     class="fi.aalto.cs.apluscourses.intellij.actions.SubmitExerciseAction"
-                    text="Submit A+ Exercise"
+                    text="Submit A+ Assignment"
                     icon="PluginIcons.A_PLUS_UPLOAD"/>
             <action id="fi.aalto.cs.apluscourses.intellij.actions.OpenSubmissionAction"
                     class="fi.aalto.cs.apluscourses.intellij.actions.OpenSubmissionAction"
                     text="Open A+ Submission"
                     icon="PluginIcons.A_PLUS_BROWSE" />
             <action id="fi.aalto.cs.apluscourses.intellij.actions.ExercisesRefreshAction"
                     class="fi.aalto.cs.apluscourses.intellij.actions.RefreshAction"
-                    text="Update Exercises"
+                    text="Update Assignments"
```

Comment:
Perhaps this PR could also change this (and Update Modules) to "Refresh x" as described in #287